content-src/lib/utils.js

```diff
@@ -7,6 +7,12 @@ module.exports = {
     const yiq = ((r * 299) + (g * 587) + (b * 114)) / 1000;
     return (yiq >= 128) ? "black" : "white";
   },
+  prettyHostname(hostname) {
+    if (!hostname) {
+      return "";
+    }
+    return hostname.toLowerCase().replace(/^www./i, "");
+  },
```

Comment:
This looks similar to the `prettyUrl()` regex below. Not sure if it makes sense to try and write a single regex/function which removes an optional "http[s]:", "//" and "www.".

```js
function prettyUrl(url) {
  if (!url) {
    return "";
  }
  return url.replace(/^((https?:)?\/\/)?(www\.)?/i, "").toLowerCase();
}
```

Then again, RegExes are an ugly thing, so maybe duplicate+simple is better than complex+ugly.

---

```
function prettyUrl(url) {
  if (!url) {
    return "";
  }
  return url.replace(/^((https?:)?\/\/)?(www\.)?/i, "").toLowerCase();
}

var sites = [
  "http://peter.com",
  "https://www.peter.com",
  "http://zombo.Com",
  "www.peter.com",
  "www.devnull.biz",
  "bleepbloop.com",
  "ftp://bananas.com",
  "//whatever.man"
];

sites.forEach((site) => {
  console.log("%s ==> %s", site, prettyUrl(site));
});